### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by gmrukwa**
I've extracted part of my work prior to release as a cleanup branch, which generally replaces project made with the Angular template into just an API one.

I've added tests to Dockerfile, since we're mostly working on Windows machines - that's the quickest way to validate them on Linux. On the other hand, image is built only on tests success during the build pipeline on Travis CI.

If some tests / docstrings are missing, they are all added in incomin review, along with all the implementations. This one is just to prune the next out of changes, that would fog the image of changes.

### CODE REVIEW

**Comment by gmrukwa** [comment]
[Cleanups](https://trello.com/c/Ej61eazJ/356-cleanups)

File: UploadApi/Controllers/DownloadController.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+*/
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Spectre.UploadApi.Services;
+
+namespace Spectre.UploadApi.Controllers
+{
+    [Route("api/[controller]")]
+    public class DownloadController : Controller
```
Comment (Sebaxp95) [inline UploadApi/Controllers/DownloadController.cs:27]: In Spectre project we have classes extends ApiController https://github.com/spectre-team/spectre/tree/master/src/Spectre/Controllers , here Controller, maybe should we use one method?
https://stackoverflow.com/questions/9494966/difference-between-apicontroller-and-controller-in-asp-net-mvc

**Comment by gmrukwa** [comment]
This is in different technology - .NET Core. I suppose we will migrate old repo for Core too.

File: UploadApi/Controllers/DownloadController.cs (hunk 1 above)
Comment (gmrukwa) [reply UploadApi/Controllers/DownloadController.cs:27]: This is in different technology - .NET Core. I suppose we will migrate old repo for Core too.

**Review by KapRom — APPROVED** [review]

**Review by Sebaxp95 — APPROVED** [review]

**Pull request merged**